digBusinessTypeExtractor/documentation_classification.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+# @Author: ZwEin
+# @Date:   2016-10-06 23:36:45
+# @Last Modified by:   ZwEin
+# @Last Modified time: 2016-10-07 00:44:15
+# -*- coding: utf-8 -*-
+# @Author: ZwEin
+# @Date:   2016-09-23 12:58:37
+# @Last Modified by:   ZwEin
+# @Last Modified time: 2016-09-24 20:29:20
+
+import os
+import sys
+import csv
+import numpy as np
+from sklearn.neighbors import NearestNeighbors
+from sklearn.neighbors import KNeighborsClassifier
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.feature_extraction.text import TfidfVectorizer
+
+
+##################################################################
+# Constant
+##################################################################
+
+DC_CATEGORY_NAMES = [
+    'unknown',
+    'others',
+    'massage',
+    'escort',
+    'job_ads'
+]
+DC_CATEGORY_DICT = {
+    'unknown': -1,
+    'others': 1,
+    'massage': 2,
+    'escort': 3,
+    'job_ads': 4
+}
+
+DATA_DIR = os.path.join(os.path.dirname(__file__), 'data')
+
+class WEDC(object):
+
+    ##################################################################
+    # Basic Methods
+    ##################################################################
+
+    def __init__(self, data_path=os.path.join(DATA_DIR, 'dataset.csv'), vectorizer_type='count', classifier_type='knn'):
```

Comment:
Hi Jason, thanks for pointing out, :). I have updated it, and please have a look for the detail, thank you